xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml

```diff
@@ -164,10 +164,12 @@ Hello 👀
   transform: rotate(90deg);
 }
 
-.confluence-expand-macro .panel-title {
+.confluence-expand-macro {
   padding: @panel-heading-padding;
 }
-
+.panel-title {
+  padding: 5px;
+}
```

Comment:
Why is this needed?

The previous rule added a padding to the `.panel-title` inside a `.confluence-expand-macro`. Now there are two css rules. One applies some padding to `.confluence-expand-macro` and one applies padding to `.panel-title`.

Additionally, for the new rule, you use an absolute value for the padding, when, according to the previous rule, you can use a LESS variable.

Reply:
A `display: block` rule should suffice in order to have the same rendition.